postmortems/002 - Postmortem for v0.0.23 release.md

```diff
@@ -0,0 +1,31 @@
+# 002 - Postmortem for v0.0.23 release
+
+## Level: Internal
+
+## Author(s)
+
+- David Ellis <david@iasql.com>
+
+## Summary
+
+We have been unable to release v0.0.23, a significant rearchitecting and feature-adding version of IaSQL, for approximately a month, *primarily* due to issues with the new infrastructure required in staging and production. There are five incidences we have observed on staging (none on production as production is not yet in use) and we have applied mitigations to staging and production for them all. This document will outline them.
+
+## Timeline
+
+- **2022-12-20**: Staging cannot load docker container images because it has run out of disk space. Increased boot volume to 16GB.
+- **2022-12-27**: Staging deploys fail complaining about lack of disk space, again. Dropping `stdout/err` in favor of LogDNA recommended.
```

Comment:
do we have a task for this already?

Reply:
It was already done in the engine, but not the run service. I did it for the run service at the same time I improved the logging and added the auto restart code.